.gitignore

```diff
@@ -15,7 +15,7 @@ backend/dist/*
 key.properties
 glob:application/custom-devices.xml
 cases/bin/
-application/custom-devices.xml
```

Comment:
Is this changing line break characters or something? Might as well remove this file from this PR.

Reply:
Not sure what happened here but this is fixed.